### PR TITLE
chore: Jenkinsfile

### PR DESCRIPTION
**Pull request by zot24**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  node {
+    stage ("Checkout") {
+      gitClean()
+      checkout scm
+    }
+
+    sshagent (credentials: ['github-moltin-moltinbot-ssh-key']) {
+      stage ("Provisioning") {
+        docker.image('node:alpine').inside {
+          npm install --production
```
Comment (ynnoj) [inline Jenkinsfile]: @zot24 Remove the `--production` flag. We need the `devDependencies` for tests.

File: Jenkinsfile [hunk 2]
```diff
@@ -28,15 +28,15 @@ try {
 
       stage ("Versioning") {
         docker.image('zot24/semantic-release-cli').inside("-v \$(pwd):/data") {
-          sh "semantic-release pre"
+          sh "semantic-release-cli pre"
```
Comment (zot24) [inline Jenkinsfile]: @ynnoj we need the `-cli` as the whole command is `semantic-release-cli`

Comment (ynnoj) [inline Jenkinsfile]: 👍

**Comment by zot24** [comment]
@ynnoj `sematic-release` fail with the following error, if you can make `sematic-release` works Jenkins will pass

screen shot 2017-05-04 at 09 35 51

I will suggest to try the following locally to see what's causing the problem

```
docker run -it --rm -v $(pwd):/data -w /data zot24/semantic-release semantic-release pre
```

I'm trying this with the G/H token now and same problem

**Pull request merged**